cli/dcoscli/marathon/main.py

```diff
@@ -293,14 +294,31 @@ def _about():
 
 def _get_resource(resource):
     """
-    :param resource: optional filename for the application or group resource
+    :param resource: optional filename or http(s) url
+    for the application or group resource
     :type resource: str
     :returns: resource
     :rtype: dict
     """
     if resource is not None:
-        with util.open_file(resource) as resource_file:
-            return util.load_json(resource_file)
+        if os.path.isfile(resource):
+            with util.open_file(resource) as resource_file:
+                return util.load_json(resource_file)
+        else:
+            try:
+                http.silence_requests_warnings()
+                req = http.get(resource)
+                if req.status_code == 200:
+                    data = b''
+                    for chunk in req.iter_content(1024):
+                        data += chunk
+                    return util.load_jsons(str(data.decode('utf-8')))
+                else:
+                    raise Exception
```

Comment:
this is done to avoid code duplication. this `Exception` catched in `excet` block and `DCOSException` raised.